docs/_src/usage/usage/languages.md

```diff
@@ -125,6 +154,21 @@ reader = FARMReader("deepset/xlm-roberta-large-squad2")
 
 <div class="tabs innertabslanguage">
 
+<div class="tabinner">
```

Comment:
Let's extend this to more languages. Can you have a quick look at https://huggingface.co/models?pipeline_tag=question-answering ? I think by now there are models for Spanish, Chinese, Korean, Portugese ...